### PR TITLE
Tip to measure performance of a javascript block

### PR DESCRIPTION
**Pull request by manmadareddy**
- using console.time and console.timeEnd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,27 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+    len = arr.length,
+    i;
+
+for (i = 0; i < len; i++) {
+    arr[i] = new Object();
+};
+console.timeEnd("Array initialize"); // Outputs: Array initialize: 0.711ms
+```
+
+more info: https://github.com/DeveloperToolsWG/console-object
```
Comment (loverajoel) [inline README.md]: change by [More info](https://github.com/DeveloperToolsWG/console-object)

**Comment by loverajoel** [comment]
Can you add a jsfiddle demo?

File: README.md [hunk 2]
```diff
@@ -18,6 +18,27 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #11 - Tip to measure performance of a javascript block
+
+2016-01-11 by [@manmadareddy]
```
Comment (loverajoel) [inline README.md]: link to twitter?

**Comment by loverajoel** [comment]
after changes, squash your commits.
Thanks

**Comment by manmadareddy** [comment]
sure I will add

**Comment by manmadareddy** [comment]
Updated my pull request with the resolved comments.

**Comment by jillesme** [comment]
Some people don't think console.time{End} is a reliable method and there are [better ways](https://mathiasbynens.be/notes/javascript-benchmarking) of measuring performance. That being said I still use console.time{End} myself but it might be worth adding that link to it as well!

File: README.md [hunk 3]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+};
+console.timeEnd("Array initialize"); // Outputs: Array initialize: 0.711ms
+```
+
+More info about console and javascript-benchmarking:
+
+https://github.com/DeveloperToolsWG/console-object
+https://mathiasbynens.be/notes/javascript-benchmarking
+
+Demo: (outputs in browser console)
```
Comment (loverajoel) [inline README.md]: Update by

Demo: [jsfiddle](https://jsfiddle.net/meottb62/) - [codepen](http://codepen.io/anon/pen/JGJPoa) (outputs in browser console)

File: README.md [hunk 4]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+
+for (i = 0; i < len; i++) {
+    arr[i] = new Object();
+};
+console.timeEnd("Array initialize"); // Outputs: Array initialize: 0.711ms
+```
+
+More info about console and javascript-benchmarking:
+
+https://github.com/DeveloperToolsWG/console-object
```
Comment (loverajoel) [inline README.md]: change by [Console object](https://github.com/DeveloperToolsWG/console-object)

File: README.md [hunk 5]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+https://mathiasbynens.be/notes/javascript-benchmarking
```
Comment (loverajoel) [inline README.md]: change by [Javascript benchmarking](https://mathiasbynens.be/notes/javascript-benchmarking)

File: README.md [hunk 6]
```diff
@@ -18,6 +18,35 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #12 - Tip to measure performance of a javascript block
+
+2016-01-12 by [@manmadareddy](https://twitter.com/manmadareddy)
+
+For quickly measuring performance of a javascript block, we can use the console functions like
+console.time(label) and console.timeEnd(label).
```
Comment (loverajoel) [inline README.md]: add link [`console.time(label)`](https://developer.mozilla.org/en-US/docs/Web/API/Console/time) and format

Comment (loverajoel) [inline README.md]: add link [`console.timeEnd(label)`](https://developer.mozilla.org/en-US/docs/Web/API/Console/timeEnd) and format

**Comment by loverajoel** [comment]
@manmadareddy ping

**Comment by manmadareddy** [comment]
sorry I missed your ping..

**Comment by manmadareddy** [comment]
@loverajoel 
Made the required changes.

**Comment by loverajoel** [comment]
@manmadareddy excellent! thanks

**Pull request merged**